coordinator-common/src/test/java/org/apache/kafka/coordinator/common/runtime/EventAccumulatorTest.java

```diff
@@ -83,15 +83,15 @@ public void testBasicOperations() {
         assertNull(accumulator.poll());
 
         List<MockEvent> events = Arrays.asList(
```

Comment:
We could use List.of here.

Reply:
I kept `Arrays.asList()` for consistency with the other test below that requires a mutable list.